runtime/altair/src/lib.rs

```diff
@@ -335,8 +334,8 @@ pub enum ProxyType {
 	Any,
 	NonTransfer,
 	Governance,
-	// Staking,
-	// Vesting,
+	_Staking,
```

Comment:
This seems like it's migrating former Staking proxies to a non-functional proxy type. It could use a comment describing what's up, and maybe a better name.